.github/workflows/run_test.yaml

```diff
@@ -1,8 +1,8 @@
 on:
   push:
-    branches: [test-debug-symbols]
+    branches: [test-debug-symbols, logging-improvements]
   pull_request:
-    branches: [test-debug-symbols]
+    branches: [test-debug-symbols, logging-improvements]
```

Comment:
is there some way to specify all branches?

Reply:
Yes, I modified the workflow such that the "R CMD check" command is run on each branch.
Fixed in commit 6b85147